samples/cpp/aruco_dict_utils.cpp

```diff
@@ -0,0 +1,270 @@
+#include <opencv2/core/hal/hal.hpp>
```

Comment:
`hal` in samples is overkill. Don't use low-level APIs in **generic samples**.

(at least because you can't wrote similar example in other languages, like Python / Java / etc)

Reply:
`cv::hal::normHamming` is used. @vovka643 Could you replace it with `cv::norm`?